src/sass/components/_product-box.scss

```diff
@@ -183,3 +183,21 @@
     }
   }
 }
+
+.chair-products, .sofa-products, .table-products, .dining-products {
```

Comment:
zapisz to proszę tak aby każdy selektor był w nowej linijce, łatwiej będzie dodawać nowe elementy lub rozwiązywać konflikty